### PR TITLE
fix:issue for term having  spaces

### PR DESCRIPTION
**Pull request by amithire1201**
### Description
https://github.com/brainstormforce/ultimate-addons-for-gutenberg/issues/422
https://wordpress.org/support/topic/taxonomy-list-really-no-links-in-blocks/
https://wordpress.org/support/topic/problems-with-taxonomy-list-block/

### Types of changes

Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?
Create taxonomy term with spaces and test the frontend

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -160,6 +160,9 @@ When you use the Ultimate Addons for Gutenberg along with the free Astra theme,
 
 == Changelog ==
 
+= 1.18.2 =
```
Comment (vrundakansara) [inline readme.txt:163]: Keep the version to = 1.18.1.1 =

File: dist/blocks/taxonomy-list/class-uagb-taxonomy-list.php [hunk 2]
```diff
@@ -531,7 +531,7 @@ function redirectToTaxonomyLink( selectedOption ) {
 		 *
 		 * @param array $attributes Array of block attributes.
 		 *
-		 * @since 1.18.1
+		 * @since x.x.x
```
Comment (vrundakansara) [inline dist/blocks/taxonomy-list/class-uagb-taxonomy-list.php:534]: Do not modify all of these.

**Review by vrundakansara — COMMENTED** [review]
@amithire1201 - Please resolve the following issues.

File: readme.txt (hunk 1 above)
Comment (amithire1201) [reply readme.txt]: Done.

File: dist/blocks/taxonomy-list/class-uagb-taxonomy-list.php (hunk 2 above)
Comment (amithire1201) [reply dist/blocks/taxonomy-list/class-uagb-taxonomy-list.php]: Done.

**Review by RajkiranBagal — APPROVED** [review]

**Review by vrundakansara — APPROVED** [review]

**Pull request merged**